book/07-extending-learners.qmd

```diff
@@ -5,47 +5,53 @@ library(mlr3)
 library(mlr3book)
 ```
 
+Although many learners are already included in the mlr3verse, there might be a situation in which your algorithm of choice is not implemented.
```

Comment:
```suggestion
Although many learners are already included in the `r mlr_pkg("mlr3verse"), there might be a situation in which your algorithm of choice is not implemented.
```

I think we don't say "mlr3verse" when we talk about the ecosystem in general, probably better to just say "mlr3" or more specifically "mlr3learners".

Reply:
will replace by mlr3 ecosystem